docs/zh/latest/plugins/openfunction.md

```diff
@@ -0,0 +1,131 @@
+---
+title: openfunction
+keywords:
+  - APISIX
+  - Plugin
+  - OpenFunction
+  - openfunction
+description: 本文介绍了关于 Apache CNCF OpenFunction 插件的基本信息及使用方法。
```

Comment:
OpenFunction is CNCF project not Apache? This expression sounds a bit strange.

Reply:
Modified.
OpenFunction is CNCF project not Apache.